spec/features/user_signs_out_spec.rb

```diff
@@ -13,12 +13,12 @@
   scenario 'authenticated user signs out' do
     user = FactoryGirl.create(:user)
 
-    visit new_user_session_path
+    visit meetups_path
 
     fill_in 'Email', with: user.email
     fill_in 'Password', with: user.password
 
-    click_button 'Log in'
+    click_button 'Sign in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.